x-pack/plugins/osquery/cypress/e2e/all/alerts.cy.ts

```diff
@@ -439,6 +439,27 @@ describe('Alert Event Details', () => {
       });
   });
 
+  it('should be able to run take action query against all enrolled agents', () => {
+    loadAlertsEvents();
+    cy.getBySel('expand-event').first().click({ force: true });
+    cy.getBySel('take-action-dropdown-btn').click();
+    cy.getBySel('osquery-action-item').click();
+    cy.getBySel('agentSelection').within(() => {
```

Comment:
selectAllAgents(); 

I would use this helper, however we may need to adjust the ''x Agents selected " text.  

What do you say?